### ISSUE TITLE
vq constants and scaling

### ISSUE THREAD

**Issue by markyoder** [geodynamics/vq#70]
this is really a TODO: in the code:

we'll want to talk about constants and scaling. note th eline in RunEvent.cpp:
double rupture_area = pow(10, -3.49+0.91*as.mag);

1) wrong place for this sort of declaration/assignment
2) we'll want to talk about scaling constants. in particular, i think this .91 should definitely be 1.0 for area, but not everybody's going to agree with me, and we want to be able to vary this, to do science and stuff.

aka: 
double area_factor = -3.5;
double b_area = 1.0;
double b_gr = 1.0;  // though, these should probably always be the same, otherwise, N/A != const... or should it?
double rupture_area = pow(10, area_factor + b_area*as.mag)

so, if we're so inclined, we can use this ticket to discuss and make requests for vq_scaling_constants.h, otherwise it will just stand as a "TODO" reminder.

**Comment by eheien** [comment]
Yes, this is a good idea.  Instead of a header, though, I'd make it a parameter so it doesn't need to be recompiled.

**Comment by kwschultz** [comment]
Current scaling parameters come from Mark Leonard's 2010 paper on scaling.
Search through the VQ source code for "Leonard 2010" to find instances where the constants are used.
